### PR TITLE
Add simplified instructions for neophytes (like me)

### PR DESCRIPTION
**Pull request by JayJohnston-WEBjuju**
I was able to get this installed.  Being quite new to plugins, it took me some time to get it running.  After getting it installed and seeing how perfectly it fits into my workflow, i wanted to (a) give back to the project, (b) prepare a relationship where i might continue to contribute, and (c) offer some instructions that would have helped me so they might help other neophytes like myself.

If by chance my instructions are in anyway wrong or could be made more clear, I look forward to editing the PR.

I did also worry a bit about where this section should go...because it is related to "Installation" i left it in the installation section; however, I realize that the elementary instructions are not as complete as those following this new section.  So there is that consideration.

Anyway, thank you for your work so far on this; i look forward to having a long relationship with this plugin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,70 @@ Plugin 'https://github.com/kezhenxu94/vim-mysql-plugin.git'
 
 Then run `:PlugInstall`.
 
+### Install For Dummies
```
Comment (kezhenxu94) [inline README.md]: What about making this a bullet list item like this?

```
## Install
- Bundle (Recommended)
- Plug (for neovim)
- Without any plugin manager 
```

File: README.md [hunk 2]
```diff
@@ -44,6 +44,70 @@ Plugin 'https://github.com/kezhenxu94/vim-mysql-plugin.git'
...
+\rs
+```
+
+Query results appear in a split pane. The following is a description of the commands including an explanation of the `\SELECT<CR>V\rs` sequence:
+
+- `/SELECT` then *<CR>* moves your cursor (via *search*) to the query
+- `V` *shift+v* selects the entire query (line)
+- `\rs` issues `<leader>`+rs
+ - earlier we set `mapleader` to backslash _(change it in .vimrc)_
+- Command reference:
```
Comment (kezhenxu94) [inline README.md]: These looks to be duplicated of the section `# Usage`? What about just adding your explanations to improve the `# Usage` section

**Review by kezhenxu94 — COMMENTED** [review]
Hi @JayJohnston-WEBjuju , thank you very much for the work, I left some comments 😇

File: README.md (hunk 1 above)
Comment (jayjohnston) [reply README.md]: Indeed!  Does linked to subheaders look okay?
install-bullet-list

File: README.md (hunk 2 above)
Comment (jayjohnston) [reply README.md]: Sure! I honestly didn't want to interfere with what was there, but tonight, i've spent some time cherry picking from what was there.  I don't think i've lost anything.  Please double check.

**Review by kezhenxu94 — APPROVED** [review]
Thank you !

**Pull request merged**